### PR TITLE
New ingress controller configuration action required message

### PR DESCRIPTION
**Pull request by robotmaxtron**

### CODE REVIEW

**Comment by robotmaxtron** [comment]
cc: @aweiteka

File: osd/Ingress-Controllers.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Ingress controller blocked, action required",
```
Comment (aweiteka) [inline osd/Ingress-Controllers.json:5]: Typical summary convention has been this pattern, "Action required: Update ingress controller configuration"

File: osd/Ingress-Controllers.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "description": "Your cluster has a configuration problem with ingress-controllers. Ingress Operator configuration documentation can be found here - https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#configuring-ingress-controller",
```
Comment (aweiteka) [inline osd/Ingress-Controllers.json:6]: Let's call out the cause and action more explicitly.

"Your cluster has a misconfigured ingress-controller. Please delete the ingress controller and create a new one that does not conflict. See documentation documentation can be found here - https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#configuring-ingress-controller",
 

 
Let's call out the cause and action more explicitly.

"Your cluster has a misconfigured ingress-controller. Please delete the ingress controller and create a new one that does not conflict. See ingress-controller documentation for details: https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#configuring-ingress-controller"

I'm not clear on the conflict part (CIDR?). Let's make this work generally for all ingress-controller issues.

**Review by aweiteka — CHANGES_REQUESTED** [review]

**Comment by robotmaxtron** [comment]
Updated the wording as requested.

**Pull request merged**